tests/openadapt/test_recording.py

```diff
@@ -0,0 +1,121 @@
+"""Module for testing the recording module."""
+
+import multiprocessing
+import time
+import os
+import pytest
+from openadapt import record, playback, utils, video
+from openadapt.config import config
+from openadapt.db import crud
+from openadapt.models import Recording, ActionEvent
+from loguru import logger
+
+RECORD_STARTED_TIMEOUT = 360  # Increased timeout to 6 minutes
+
+
+@pytest.fixture
+def setup_db():
```

Comment:
This appears to be unused. If so, can you please remove it?

Reply:
Yeah

Reply:
Done